app/forms/select_email_form.rb

```diff
@@ -7,6 +7,9 @@ class SelectEmailForm
   attr_reader :user, :identity, :selected_email_id
 
   validate :validate_owns_selected_email
+  validates :selected_email_id, presence: {
+    message: proc { I18n.t('email_address.not_found') },
```

Comment:
I'd suggest using the generic "This field is required" for the presence check, rather than "Email address not found"

```suggestion
    message: proc { I18n.t('simple_form.required.text') },
```